src/providers/ldap/sdap.c

```diff
@@ -1863,6 +1867,26 @@ sdap_get_primary_fqdn(TALLOC_CTX *mem_ctx,
         goto done;
     }
 
+    /* In AD scenarion, the object can be from subdomain - identify it by SID */
+    if (sid_attr_name != NULL) {
+        ret = sdap_attrs_get_sid_str(
+            tmp_ctx,
+            idmap_ctx,
```

Comment:
Hi,

the indentation here is a bit odd, can you align the arguments in the column after '(' in the starting line?

bye,
Sumit

Reply:
Thanks for spotting this, updated

Reply:
Thanks.